src/main/kotlin/pt/up/fe/ni/website/backend/service/ErrorMessages.kt

```diff
@@ -44,6 +44,8 @@ object ErrorMessages {
 
     fun roleNotFound(id: Long): String = "role not found with id $id"
 
+    fun photoNotFound(): String = "photo not found"
```

Comment:
I think it would be beneficial to include the image's name here, and perhaps make it more general by using "Image" instead of "Photo"